frontend/src/app/home.tsx

```diff
@@ -64,38 +65,56 @@ function Homepage() {
 
   const handleSubmit = () => {
     if (state.modelURL && state.datasetURL) {
-      console.log(`Input 1: ${state.modelURL}, Input 2: ${state.datasetURL}`);
-      console.log(`Selected Metrics: ${state.metricChips.filter((metricChip) => metricChip.selected).map((metricChip) => metricChip.label).join(", ")}`);
-      console.log(checkURL(state.modelURL) && checkURL(state.datasetURL))
-      
-      const user_info = {"modelURL": state.modelURL, 
+      const user_info = {
+        "modelURL": state.modelURL, 
         "datasetURL": state.datasetURL,
-        "metrics": state.metricChips.filter((metricChip) => metricChip.selected).map((metricChip) => metricChip.label)
-      }
-      
-      // Create the text content for the file
-      const textContent = `
-      AIgnostic Report
-      ===================
-      Model API URL: ${user_info.modelURL}
-      Dataset API URL: ${user_info.datasetURL}
-      Selected Metrics: ${
-        user_info.metrics.length > 0
-          ? user_info.metrics.join(", ")
-          : "No metrics selected"
-      }
-    `;
+        "metrics": state.metricChips.filter((metricChip) => metricChip.selected)
+                                    .map((metricChip: any) => (metricChip.label).toLowerCase())
+      };
+
+      // send POST request to backend server
+      fetch(BACKEND_URL, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(user_info),
+      })
+        .then((response) => {
+          if (!response.ok) {
+            throw new Error(`HTTP error! status: ${response.status}`);
+          }
+          return response.json();
+        })
+        .then((data) => {
+          const results = data["results"]
+          // Create the text content for the file
+          const textContent = "AIgnostic Report" + "\n" +
```

Comment:
There's an argument about using a JS template string here but formatting wouldn't be great